packages/theme/components/Header/Navigation/__tests__/HeaderNavigationSubcategories.spec.js

```diff
@@ -0,0 +1,54 @@
+import userEvent from '@testing-library/user-event';
+import HeaderNavigationSubcategories from '../HeaderNavigationSubcategories.vue';
+import { render } from '~/test-utils';
+import CategoryTreeDataMock from '~/test-utils/mocks/categoryTreeDataMock';
+import { useUiHelpers } from '~/composables';
+import useUiHelpersMock from '~/test-utils/mocks/useUiHelpersMock';
+
+jest.mock('~/composables');
+useUiHelpers.mockReturnValue(useUiHelpersMock());
+
+// Women category selection
+const childrenProps = { props: { currentCategory: CategoryTreeDataMock[0].children[1] } };
+const noChildrenProps = { props: { currentCategory: CategoryTreeDataMock[0].children[0] } };
+
+describe('HeaderNavigationSubcategories', () => {
+  it('displays level 1 children category links', () => {
+    const { getByRole } = render(HeaderNavigationSubcategories, childrenProps);
+    getByRole('link', { name: /tops/i });
+    getByRole('link', { name: /bottoms/i });
+  });
+
+  it('displays level 2 children category links', () => {
+    const { getByRole } = render(HeaderNavigationSubcategories, childrenProps);
+    getByRole('link', { name: /jackets/i });
+    getByRole('link', { name: /hoodies & sweatshirts/i });
+    getByRole('link', { name: /tees/i });
+    getByRole('link', { name: /tanks/i });
+  });
+
+  it('do not display any links if root category has no children', () => {
+    const { queryByTestId } = render(HeaderNavigationSubcategories, noChildrenProps);
+    expect(queryByTestId(/navigation-subcategories/i)).toBeNull();
```

Comment:
```suggestion
    expect(queryByTestId('navigation-subcategories')).toBeNull();
```
As far as I've seen in the code you use `data-testid='navigation-subcategories'` not `:data-testid="'navigation-subcategories' + somePrefix"`, so the regex is not necessary

Reply:
Ok, fixed.